src/scss/components/header.scss

```diff
@@ -108,34 +108,54 @@ body > header {
 }
 
 // Main site header search
-.site-header-search {
-  .form-control,
-  button[type="submit"] {
-    @include form-controls-style();
-    border: 1px solid rgba(255, 255, 255, 0.5);
-
-    // Special case for white/light-grey theme
-    body.theme-header-white &,
-    body.theme-header-light-grey & {
-      background: $input-bg;
-      border-color: $input-border-color;
-    }
+@include media-breakpoint-up(sm) {
```

Comment:
This isn't centered vertically

![image](https://user-images.githubusercontent.com/111025/62337702-db5b6f80-b529-11e9-92d9-1b74dd374b85.png)

Reply:
Horizontally? Fixed.